cmd/root.go

```diff
@@ -133,9 +133,10 @@ var getManifestsCmd = &cobra.Command{
 		}
 		manifests, err := client.GetManifests()
 		if err != nil {
-			klog.Fatal(err)
+			color.Red(err.Error())
+			os.Exit(1)
```

Comment:
Doing this to match the behavior of python reckoner. Our tests expect exit code 1 and `klog.Fatal()` exits with code 255